### PR TITLE
Update 200.0.0 references in sample

### PR DESCRIPTION
**Pull request by prupani-7**
## Description

Update references of "beta"/ 200.0 from the readme files and point links to point to the updated 200.1 pages.

## Links and Data

Sample Epic: `runtime/kotlin/issues/2090 `

### CODE REVIEW

**Comment by prupani-7** [comment]
@shubham7109 could you do a review on this PR?

File: README.md [hunk 1]
```diff
@@ -17,29 +20,37 @@ Once the project is cloned to disk you can import into Android Studio:
...
 
 ## Accessing Esri location services
+
 Accessing Esri location services, including basemaps, routing, and geocoding, requires authentication using either an API Key or an ArcGIS identity:
- 1. API key: A permanent key that gives your application access to Esri location services. Visit your [ArcGIS Developers Dashboard](https://developers.arcgis.com/dashboard) to create a new API key or access an existing API key.
- 
-The Android samples in this repository have been structured to use an API key, set once, which will run in all samples. Set your API key in the `gradle.properties` file located in the `/.gradle` folder within your home directory (`/Users/<user_name>/.gradle/gradle.properties`). The API_KEY property should contain quotes around the key itself: `API_KEY = "YOUR_API_KEY"`
 
-2. ArcGIS identity: An ArcGIS named user account that is a member of an organization in ArcGIS Online or ArcGIS Enterprise.
+* API key
```
Comment (shubham7109) [inline README.md:26]:
```suggestion
#### API key
```
Since the PR here makes some header changes, would be nice to use the header here as well.

File: README.md [hunk 2]
```diff
@@ -17,29 +20,37 @@ Once the project is cloned to disk you can import into Android Studio:
...
+
+A permanent key that gives your application access to Esri location services. Visit your [ArcGIS Developers Dashboard](https://developers.arcgis.com/dashboard) to create a new API key or access an existing API key.
+The Android samples in this repository have been structured to use an API key, set once, which will run in all samples.
+Set your API key in the `gradle.properties` file located in the `/.gradle` folder within your home directory (`/Users/<user_name>/.gradle/gradle.properties`).
+The API_KEY property should contain quotes around the key itself: `API_KEY = "YOUR_API_KEY"`
+
```
Comment (shubham7109) [inline README.md:32]:
```suggestion
Set your API key in the `gradle.properties` file located in the `/.gradle` folder within your home directory.
The API_KEY property should contain quotes around the key itself: 

```
Minor suggestion:
I think pulling out the `gradle.properties` example as a code block would help better clarify the usage for API key. Could add something like this below the API key description:
```gradle
API_KEY = "YOUR_API_KEY" // path: /Users/<user_name>/.gradle/gradle.properties
```

File: README.md [hunk 3]
```diff
@@ -17,29 +20,37 @@ Once the project is cloned to disk you can import into Android Studio:
...
+* ArcGIS identity
```
Comment (shubham7109) [inline README.md:33]:
```suggestion
#### ArcGIS identity
```
Could be a header here as well.

File: README.md [hunk 4]
```diff
@@ -17,29 +20,37 @@ Once the project is cloned to disk you can import into Android Studio:
...
+
 You can execute all the build tasks using the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) command line tool. It's available as a batch file for Windows (gradlew.bat) and a shell script for Linux/Mac (gradlew.sh) and it is accessible from the root of the project.  
 
-- Build a debug APK
+* Build a debug APK
 
-```
-$ ./gradlew assembleDebug
+```gradle
+./gradlew assembleDebug
```
Comment (shubham7109) [inline README.md:48]: The code block is not `.gradle` code but just a terminal execution.

File: README.md [hunk 5]
```diff
@@ -65,12 +76,10 @@ Anyone and everyone is welcome to [contribute](.github/CONTRIBUTING.md). We do a
...
 ## Licensing
-Copyright 2022 Esri
-
-Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
-
-http://www.apache.org/licenses/LICENSE-2.0
 
+Copyright 2023 Esri
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+You may obtain a copy of the License <http://www.apache.org/licenses/LICENSE-2.0>
```
Comment (shubham7109) [inline README.md:82]: Revert these changes as the copyright is not planned to be updated. I believe the last discussion was to use the same copyright year at the time of the creation of the sample.

**Review by shubham7109 — CHANGES_REQUESTED** [review]
Few comments here, thanks @prupani-7

File: README.md (hunk 4 above)
Comment (prupani-7) [reply README.md:48]: this was to resolve a warning from code inspection. to have the language specified. My intention of adding `gradle` was to specify that its a `gradle command` for the terminal. Correct, its not a language, but then I dont know what else suits best here so we can address the warning.

File: README.md (hunk 5 above)
Comment (prupani-7) [reply README.md:82]: Interesting! But our 200.1.0 samples use the 2023 license right?

Comment (prupani-7) [reply README.md:82]: Also, line 82 address code inspection warning on using http links in the .md files.

File: README.md (hunk 4 above)
Comment (shubham7109) [reply README.md:48]: - Are you referring to "Inspect code"? Don't think we should modify non-code files using inspect code in the first place. 
- If you are referring to the GitHub [README_style_checker](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/main/tools/CI/README_Metadata_StyleCheck/README_style_checker.py), the warnings are designed to format the readme file for a new sample, not the top-level repository readme. 

In either case, if the code block has a language that can use [style highlighting](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) it should be used. This code block does not contain gradle code and probably needs to be removed.

Comment (prupani-7) [reply README.md:49]: I was referring to GitHub [README_style_checker](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/main/tools/CI/README_Metadata_StyleCheck/README_style_checker.py), sorry for the typo.

Comment (prupani-7) [reply README.md]: I reverted it based off your comment. 
But I don't get why we shouldn't resolve the warnings by the Github style checker for the top level Readme (vs samples Readme) as it's still part of the samples repo. I think it would be good if we are able to adhere to the Readme guidelines in general for the repo :)

Comment (shubham7109) [reply README.md]: Sure, resolving obvious warnings is a no-brainer! And the heading changes in the PR are welcomed. To me, the expected highlighting style isn't that obvious from the [list](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) of accepted keywords. If possible resolving them is ideal, but the code block does serve its function as it is.

File: README.md (hunk 5 above)
Comment (shubham7109) [reply README.md]: > But our 200.1.0 samples use the 2023 license right?

Not quite, samples added to the repo in 2022 use `Copyright 2022` and samples added to the repo in 2023 use `Copyright 2023`. Or at least, they should be 😅 

> Also, line 82 addresses GitHub [README_style_checker](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/main/tools/CI/README_Metadata_StyleCheck/README_style_checker.py), warning on using http links in the .md files.

As regards the licensing, there are ESRI guidelines on the format of licensing. You can visit any platform samples repo to notice a `LICENSE` file and the license text at the bottom. In fact, should see in all public ESRI repositories. The GitHub checker is helpful, but in order to change this text it probably will need approval first.

Comment (prupani-7) [reply README.md]: The last line makes sense, good to know.

**Comment by prupani-7** [comment]
@shubham7109 I have addressed your feedback. Please review and let me know if its look good to be merged? Thanks!

**Review by shubham7109 — APPROVED** [review]
Looks good 👍
